docs/main/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/openebs-on-kubernetes-platforms/gke.md

```diff
@@ -0,0 +1,259 @@
+---
+id: gke
+title: Replicated PV Mayastor Installation on Google Kubernetes Engine
+keywords:
+ - Replicated PV Mayastor Installation on Google Kubernetes Engine
+ - Replicated PV Mayastor Installation on GKE
+ - Replicated PV Mayastor - Platform Support
+ - Platform Support
+ - Google Kubernetes Engine
+ - GKE
+description: This section explains about the Platform Support for Replicated PV Mayastor.
+---
+# Replicated PV Mayastor Installation on Google Kubernetes Engine
+
+This document provides instructions for installing Replicated PV Mayastor on Google Kubernetes Engine (GKE).
```

Comment:
We need a better messaging here, as to why one should use Mayastor on GKE. Check this blog: https://blog.openebs.io/building-persistent-storage-with-openebs-and-local-ssds-on-gke-for-stateful-applications-87df6880188f

Reply:
Please don't put the blog link in this doc. It's outdated but the messaging is good.